kong-ngx-build

```diff
@@ -387,6 +393,70 @@ main() {
   succ "Build finished in $SECONDS seconds. Enjoy!"
 }
 
+parse_version() {
+  [[ -z $1 || -z $2 ]] && return 1
```

Comment:
We should probably throw a hard error here, as this signifies a programmer error.